PowerShell.sln

```diff
@@ -159,6 +153,14 @@ Global
 		{73EA0BE6-C0C5-4B56-A5AA-DADA4C01D690}.Debug|Any CPU.Build.0 = Linux|Any CPU
 		{73EA0BE6-C0C5-4B56-A5AA-DADA4C01D690}.CodeCoverage|Any CPU.ActiveCfg = Linux|Any CPU
 		{73EA0BE6-C0C5-4B56-A5AA-DADA4C01D690}.CodeCoverage|Any CPU.Build.0 = Linux|Any CPU
+		{190BB016-1395-4944-853F-98F8A4277483}.CodeCoverage|Any CPU.ActiveCfg = Release|Any CPU
```

Comment:
Can you please explain why these changes to ConfigurationPlatforms are necessary?